infrastructure/display-api-service/gitconfig

```diff
@@ -0,0 +1,9 @@
+[color]
```

Comment:
Actually only the "[safe]" config is required to make things work.

Reply:
Will leave in just the same. Can see we do stuff like `%C(yellow)%d%Creset %s %C(bold blue)<%an>%Creset`